helm/cluster-api/files/bootstrap/patches/versions/v1alpha3/kubeadmconfigs.bootstrap.cluster.x-k8s.io.yaml

```diff
@@ -379,7 +379,7 @@
                         type: array
                       token:
                         description: Token is used for establishing bidirectional trust between nodes and control-planes. Used for joining nodes in the cluster.
-                        type: object
+                        type: string
```

Comment:
This was maybe updated in newer 0.3.x version, as the current one is not the latest, will check.

Reply:
We are currently not using v1alpha3 version of bootstrap types, so ignoring this.